sdk/storage/azure-storage-blob/v10_v12.md

```diff
@@ -0,0 +1,364 @@
+# Storage Blob Service SDK Migration Guide from 10.x/11.x to 12.x
+
+In this section, we list main changes you need to be aware of when converting your Storage Blob SDK library from Version 10 to Version 12.
+## Prerequisites
+Java Development Kit (JDK) with version 8 or above
+
+## Update reactive framework from RxJava to Reactor and ReactorNetty
+For more info related to Reactor project, please refer [Reactor Project](https://projectreactor.io/)
+For more info of the motivation to V12, please refer [V12 Upgrade Storage](https://github.com/Azure/azure-storage-java/blob/master/V12%20Upgrade%20Story.md)
```

Comment:
I would be consistent about using either `v12` or `V12` and the same for the 10/11. This uses the capitalised version, whereas the table below uses a lowercase.